### PR TITLE
Overhaul install instructions

### PR DESCRIPTION
**Pull request by ISSOtm**
I took a second look at them following someone's confusion on IM, and I have to agree they were not great. I think this is a noticeable improvement.

### CODE REVIEW

File: src/pages/install/master.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+As this is highly OS-specific, we cannot provide any generic instructions.
+Sorry!
+
+## Building it yourself
+
+:::tip
+
+If your package manager is not an option, this is the preferred way.
+Unless you are on Windows, because development on Windows is hell; consider [using our CI](#using-our-ci) instead.
```
Comment (avivace) [inline src/pages/install/master.md:24]:
```suggestion
Unless you are on Windows, consider [using our CI](#using-our-ci) instead.
```
I would avoid harsh/sarcastic comments here, they really don't help anyone

**Review by avivace — CHANGES_REQUESTED** [review]

File: src/pages/install/master.md (hunk 1 above)
Comment (ISSOtm) [reply src/pages/install/master.md:24]: I've had another series of bad times with Windows yesterday :upside_down_face:

Comment (pinobatch) [reply src/pages/install/master.md]: avivace's suggested change inverts the meaning. It was "If Windows then use our CI." The suggested change changes it to "If *not* Windows then use our CI." Try "Unless you are on Windows, in which case consider...".

File: src/pages/install/source.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+ - `bison` (or `yacc`/`byacc` prior to RGBDS 0.5.0)
+ - `pkg-config` (unless using `cmake`)
+
+If you are building RGBGFX, the following are also required:
+
+ - A C++ compiler, typically `g++` or `clang`
+ - `libpng`'s development headers. For Linux users: the package is typically called [`libpng-dev`](https://packages.ubuntu.com/focal/libpng-dev), [`libpng-devel`](https://software.opensuse.org/package/libpng16-devel-64bit), or sometimes plainly [`libpng`](https://www.archlinux.org/packages/extra/x86_64/libpng/). Additionally, a `16` or `1.6` may be present after `libpng` ([example](https://tracker.debian.org/pkg/libpng1.6)).
+
+Two build systems are supported: plain old `make`, and CMake.
+Both can be used on most systems, but Windows users will almost certainly be unable to use `make`.
```
Comment (lesleyrs) [inline src/pages/install/source.md:59]: This is false, 'make' can be used on Windows. I've been using it from here https://github.com/skeeto/w64devkit

I'm trying rgbds on Windows right now so I don't have to waste memory on wsl, and it seems to work other than the one liner command from here https://rgbds.gbdev.io/docs/v0.6.1/rgbds.7 it seems to fail due to the pipes.

What was your specific problem with Windows @ISSOtm? I know there are many problems though lol

**Review by lesleyrs — CHANGES_REQUESTED** [review]

File: src/pages/install/source.md (hunk 2 above)
Comment (lesleyrs) [inline src/pages/install/source.md:59]:
```suggestion
```

Comment (lesleyrs) [reply src/pages/install/source.md:59]:
```suggestion
Two build systems are supported: plain old `make`, and CMake.
```

Comment (ISSOtm) [reply src/pages/install/source.md:59]: Yes, I know `make` can be used on Windows, hence the "almost". The second line was added after several Windows users came confused as to which they should pick, and/or after struggling to get Make *and* the corollary-required Coreutils.

Comment (zlago) [reply src/pages/install/source.md:59]: > I'm trying rgbds on Windows right now so I don't have to waste memory on wsl, and it seems to work other than the one liner command from here https://rgbds.gbdev.io/docs/v0.6.1/rgbds.7 it seems to fail due to the pipes.

dont you "just" need to use bash for that?

Comment (lesleyrs) [reply src/pages/install/source.md:59]: > dont you "just" need to use bash for that?

I said pipes but I was actually referring to whatever the '-' dashes are, it doesn't seem to work on both git bash and cmd/powershell. `error: -: Cannot read node #0's iter #43: Unexpected end of file`. Changing line endings doesn't seem to make it work either.

Comment (ISSOtm) [reply src/pages/install/source.md:59]: `-` instructs to use stdin/out, which should indeed be pipes. Though Windows tends to mangle binary data going through those, yay!

File: src/pages/install/master.md (hunk 1 above)
Comment (avivace) [reply src/pages/install/master.md]: Right, didn't really re read - my point there is avoid those kind of remarks

Comment (ISSOtm) [reply src/pages/install/master.md]: Part of the intention was to justify why Windows requires heading a different way. What's suggested front-and-center is "compile it yourself", which is a massive PITA on Windows, and I think warning of that upfront is preferable to the user finding out about all that friction on their own, and giving up.

Comment (Lokathor) [reply src/pages/install/master.md]: github CI can do windows builds can't it?

Comment (ISSOtm) [reply src/pages/install/master.md]: Yes, that's exactly what the paragraph is directing readers towards.

Comment (Lokathor) [reply src/pages/install/master.md]: Sorry, the start of the sentence made me think that Windows *is not* supported by the CI builds

Comment (ISSOtm) [reply src/pages/install/master.md]: Yeah, as Pino mentioned, it's now backwards, lol

**Pull request merged**